examples/benchmark.py

```diff
@@ -54,6 +67,20 @@ def runOneTest(testName, options):
                 f.setForceGroup(1)
         dt = 0.002*unit.picoseconds
         integ = mm.MTSIntegrator(dt, [(0,2), (1,1)])
+    elif amber:
+        dir = downloadAmberSuite()
+        prmtop = app.AmberPrmtopFile(os.path.join(dir, f'PME/Topologies/{testName}.prmtop'))
+        inpcrd = app.AmberInpcrdFile(os.path.join(dir, f'PME/Coordinates/{testName}.inpcrd'))
+        topology = prmtop.topology
+        positions = inpcrd.positions
+        dt = 0.004*unit.picoseconds
+        method = app.PME
+        cutoff = options.cutoff
+        constraints = app.HBonds
+        hydrogenMass = 3*unit.amu # NOTE do the inputs already have this included?  3.0 is standard for tleap
+        friction = 1*(1/unit.picoseconds)
+        integ = mm.LangevinMiddleIntegrator(300*unit.kelvin, friction, dt)
+        system = prmtop.createSystem(nonbondedMethod=method, nonbondedCutoff=cutoff, constraints=constraints, hydrogenMass=hydrogenMass)
```

Comment:
You shouldn't specify `hydrogenMass` since the `prmtop` file already contains heavy hydrogens. This would further alter hydrogen masses, leading to problems.

Reply:
Okay - fixed to just use the masses from the file.